### PR TITLE
Share previews

### PR DESCRIPTION
**Pull request by stanislav-slavov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build:stats": "vue-cli-service build --report-json",
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "vue-cli-service test:e2e",
-    "lint:vue": "vue-cli-service lint --no-fix",
+    "lint:vue": "vue-cli-service lint --fix",
```
Comment (CedrikNikita) [inline package.json:10]:
```suggestion
    "lint:vue": "vue-cli-service lint --no-fix",
```

Comment (stanislav-slavov) [reply package.json:10]: This is just a testing purpose PR, these change won't be part of it

Comment (CedrikNikita) [reply package.json:10]: This is marked as a PR not a draft, that's why i have add this comment.

Comment (stanislav-slavov) [reply package.json:10]: No problem, I haven't marked it as draft because I needed the deployment link after every commit.

**Pull request closed without merging**